### DISCUSSION TITLE
interested in using LIAP, but how is the support?

### DISCUSSION THREAD

**Discussion by cwilde1** [imdhemy/laravel-in-app-purchases#347] [Q&A]
I'm interested in using LIAP, but how is the support?  I've read over the project documentation several times, but I see some unanswered questions. I am ready to hire a developer to implement this for a new project, and getting support is very important to me.

For example, one question I have is: 
On this page: https://imdhemy.com/laravel-iap-docs/docs/google-play/testing-and-mocking

 at the bottom is this statement:

// THE FOLLOWING LINE SHOULD BE IN YOUR CLIENT CODE
    $receipt = Subscription::googlePlay($client)->id($itemId)->token($token)->get();
    
When you say client, are you talking about the server code we have running that's serving as the endpoint for the validation? Since this is PHP, did you mean the server code running on our server?

Thanks,
Clint

**Comment by imdhemy** [comment]
Hi @cwilde1 
Thanks for your interest in LIAP. I'm answering questions as soon as possible, and I update the documentation when I find time for that, and I'm sure everyone used it is happy :) 

Regarding your questions about the client code. That comment is meant to indicate that the following code should be in the actual source code not the test code, so yes a server side code.

**Reply by cwilde1** [reply]
Ok, thanks for the response @imdhemy.   That helps.  We are starting the integration soon and I'm glad to get your response.

**Comment by nxtboyIII** [comment]
Hi @imdhemy,

We have the product verification working now I believe.   Thank you.  It's working great so far.  Does your API also support acknowledgment?  I understand Google requires a product acknowledgment after the purchase via this api here:

https://developers.google.com/android-publisher/api-ref/rest/v3/purchases.products/acknowledge

Thanks again!
Clint

**Reply by imdhemy** [reply]
Hi @nxtboyIII 
Glad to hear that. Yes, the `Product::googlePlay()` and the `Subscription::googlePlay()` support acknowledge.

Both should used the same way of [verification](https://imdhemy.com/laravel-iap-docs/docs/google-play/sell-subscriptions), but `acknowledge()` instead of `get()`.
